rest/src/db/dbUtils.js

```diff
@@ -83,12 +83,27 @@ const dbUtils = {
 	},
 
 	/**
-	 * Convert Address to Base32 format.
-	 * @param {MongoDb.Binary} addressBinary Address from MongoDb.
-	 * @returns {string} Address in Base32 format.
+	 * Convert binary to resolved address
+	 * @param {MongoDb.Binary} binary Address|NamespaceId from MongoDb.
+	 * @returns {string} AddressBase32|NamespaceId
 	 */
-	bufferToAddressBase32: addressBinary => {
-		const hexToUint8 = convert.hexToUint8(addressBinary.toString('hex'));
+	bufferToResolvedAddress: binary => {
```

Comment:
the name would be `bufferToUnresolvedAddress`. The function is returning an Encoded Address or a Mosaic Id Hex.